packages/scss/src/components/card/mods.scss

```diff
@@ -20,13 +24,24 @@
 }
 
 @mixin elevated {
-	box-shadow: var(--commons-elevations-elevation-1);
+	box-shadow: var(--commons-boxShadow-S);
 	border: 0;
 }
 
-@mixin clickableElevated {
-	&:focus-visible,
+@mixin action {
+	box-shadow: var(--commons-boxShadow-S);
+	border: 0;
+	cursor: pointer;
+	display: flex;
+	flex-direction: row;
```

Comment:
Row is the default value, do we need to define the property?

Reply:
Nop, it's related to my first approach where the default card has a `flex-direction: column`.
Fixed.